src/forms/common.ts

```diff
@@ -0,0 +1,14 @@
+export interface SpaceSchema {
```

Comment:
shouldn't this file be called `commonSpaceSchema` ?

Reply:
It's a schema for the interface Space, it isn't a schema for a field that might happen on multiple. 
I just added the common as a moniker to shared resources between the form validation schemas/functions.

Either way, more changes can be done later, this one should probably be less limited as to not cause too many merge conflicts.